suite/tests/api/ir_x86.c

```diff
@@ -1261,6 +1261,23 @@ test_regs(void *dc)
     ASSERT(reg == DR_REG_RAX);
 #endif
 
+#ifdef X86
```

Comment:
This is not needed, the test already is x86 only. Sorry for not seeing this earlier. Could you also please remove the unnecessary #ifdef X86 that was added with 92e3619fd, and note this in the commit message? Thanks!

Reply:
Done - I'll push the other fix after the tests not to clog up resources.

Reply:
Thanks for checking this PR. PTAL